towerlib/entities/host.py

```diff
@@ -280,17 +280,30 @@ def recent_jobs(self):
         """
         return self._data.get('summary_fields', {}).get('recent_jobs')
 
+    @property
+    def job_events(self):
+        """The most job events run on the host.
+
+        Returns:
+            list if dict: The job events run on the host.
+
+        """
+        url = self._data.get('related', {}).get('job_events')
+        return EntityManager(self._tower,
+                             entity_object='JobEvent',
+                             primary_match_field='job',
```

Comment:
This field does not make any sense as a matching field as it is an object and cannot be used for the filtering method.

Reply:
Removed